scripts/react-native-cli-helper.js

```diff
@@ -5,7 +5,10 @@ const common = require('./common')
 const fs = require('fs')
 
 module.exports = {
-  buildAndroid: function buildAndroid (sourceFixtures, destFixtures) {
+  buildAndroid: function buildAndroid (sourceFixturesIn, destFixturesIn) {
+    const baseDir = process.env.PWD
```

Comment:
This made it a little easier to be agnostic of whether it was being run locally or in a Docker container on CI.